geometries/latlon/latlon_algorithm.py

```diff
@@ -32,7 +32,7 @@ def latlon(utc,target):
 
     # # # # # PART 3: FIND RA/DEC OF ASPERA WRT EARTH # # # # #
 
-    [range, ra_rad, dec_rad] = sp.recrad(pos_ITRF93)
+    [radius, ra_rad, dec_rad] = sp.recrad(pos_EARTH) #btc keep range()
```

Comment:
Assigning a value to a variable named **range** will override Python's built-in range function.  Doing so may not matter here, but it's still not a best practice.